reference/docs-conceptual/security/remoting/SSH-Remoting-in-PowerShell.md

```diff
@@ -70,67 +70,65 @@ remote computer. And, you must enable **password** or **key-based** authenticati
-   ```
+     ```
+     Subsystem powershell C:/progra~1/powershell/7/pwsh.exe -sshs
 
-   > [!NOTE]
-   > Starting in PowerShell 7.4, you no longer need to use the `-NoLogo` parameter when running
-   > PowerShell in SSH server mode.
+     > [!NOTE]
+     > There is a bug in OpenSSH for Windows that prevents you from using a path with spaces for the
+     > subsystem executable. There are two ways to work around this issue:
+     >
+     > - Use the Windows _8.3-style_ short name for the PowerShell executable path
+     > - Create a symbolic link to the PowerShell executable that results in a path without spaces
+     >
+     > For more information, see this [GitHub issue][09].
 
-   > [!NOTE]
-   > The full path to the PowerShell executable can be found by running:
-   >
-   > ```powershell
-   > (Get-Command pwsh).Source
-   >```
-   > 
-   > Due to a bug in OpenSSH for Windows that prevents spaces from working in subsystem executable
-   > paths, you must use a path containing no whitespace.\
-   >  For more information, see this [GitHub issue][09].
-   >
-   > One option is to create a symbolic link to the executable:
-   >
-   > ```powershell
-   > New-Item -ItemType SymbolicLink -Path C:\ProgramData\ssh\ -Name pwsh.exe -Value (Get-Command pwsh.exe).Source
-   > ```
-   >
-   > This places a symbolic link to the PowerShell executable in the same directory used by the OpenSSH server to
-   > store the host keys and other configuration. The link can be updated if the path to the executable ever changes,
-   > without also needing to update your sshd_config file.
-   >
-   > If you are unable to use a symbolic link, another option is to use DOS "8.3"-style short names for the components
-   > of the path to the PowerShell executable. This requires that the legacy NTFS 8dot3name file system option is
-   > enabled for the system and for the volume on which PowerShell is installed, and an 8.3 name must exist for all
-   > path components. It is inadvisable to enable that option if it is not currently enabled, and doing so may be
-   > restricted by group policy.
-   >
-   > You can use the following command to get the full 8.3 path to pwsh.exe:
-   >
-   > ```powershell
-   > cmd.exe /Q /C "for %I in (`"$((Get-Command pwsh.exe).Source)`") do echo %~sI"
-   > ```
-   >
-   > Example output on a system with PowerShell 7 Preview installed (may not be identical on all systems):
-   > ```Output
-   > C:\PROGRA~1\POWERS~1\7-PREV~1\pwsh.exe
-   > ```
-
-   Optionally, enable key authentication:
+     You only need to get the 8.3-style name for the segment of the path that contains the space. By
+     default PowerShell 7 is installed in `C:\Program Files\PowerShell\7\`. The 8.3-style name for
+     `Program Files` should be `progra~1`. You can use the following command to verify the name:
 
-   ```
-   PubkeyAuthentication yes
-   ```
+     ```powershell
+     Get-CimInstance Win32_Directory -Filter 'Name="C:\\Program Files"' |
```

Comment:
My thinking on replacing this command, specifically, was that the proposed command works no matter where it is installed, including non-standard paths. This only covers the case of it being installed in `C:\Program Files`, specifically. 🤷‍♂️